src/bonefish/broker/wamp_broker.cpp

```diff
@@ -235,7 +235,7 @@ void wamp_broker::process_unsubscribe_message(const wamp_session_id& session_id,
 
         auto topic_subscriptions_itr = m_topic_subscriptions.find(topic);
         if (topic_subscriptions_itr == m_topic_subscriptions.end()) {
-            BONEFISH_TRACE("error: broker topic subscription out of sync");
+            BONEFISH_ERROR("error: broker topic subscription out of sync");
```

Comment:
nit: make sense add `error:` prefix to macro definition and not copy it

Reply:
I tried to avoid any big changes and leave messages intact, so just split macro in two